.travis.yml

```diff
@@ -1,70 +1,53 @@
+os: linux
 dist: jammy
-addons:
-  apt:
-    update: true
-    packages: [build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libreadline-dev libffi-dev libsqlite3-dev wget libbz2-dev]
+language: python
+python:
+- "3.10"
+- "3.9"
+- "3.8"
+- "3.7"
```

Comment:
The downside to this is that it puts us on the Travis maintainer's release schedule. The versions of python that are currently available on a worker are quite a bit behind.

FWIW, I'd verified that Travis was doing a good-enough job caching pyenv builds. We should discuss with the team whether we want to make this change.

Reply:
I don't think it's that important unless we actually do find a problem with some minor release (and if that happens we can always just point-test that one in a separate and short build step). I think it makes more sense to use travis-provided pythons — it cuts down build time, drastically simplifies the CI configuration and enables build parallelism (sure, we make our own version matrix, but that complicates it even further). It's also more robust (as the build being suddenly broken — which led me to do this change — shows). I don't think we really need to live on the bleeding edge.

Reply:
ok